From 09cbafd54d80a77e9a03c1db9ea86007f9eb5da9 Mon Sep 17 00:00:00 2001
From: Mathieu Westphal <mathieu.westphal@kitware.com>
Date: Sun, 30 Apr 2023 14:06:21 +0200
Subject: [PATCH] Libraries in link order with coloring support

---
 adm/templates/OpenCASCADEConfig.cmake.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/adm/templates/OpenCASCADEConfig.cmake.in b/adm/templates/OpenCASCADEConfig.cmake.in
index e391ddfe..f37b38cf 100644
--- a/adm/templates/OpenCASCADEConfig.cmake.in
+++ b/adm/templates/OpenCASCADEConfig.cmake.in
@@ -50,8 +50,8 @@ set (OpenCASCADE_CXX_FLAGS    "@CMAKE_CXX_FLAGS@")
 # List of available OpenCASCADE modules.
 set (OpenCASCADE_MODULES @OCCT_MODULES_ENABLED@)
 @SET_OpenCASCADE_MODULES_TOOLKITS@
-# List of available OpenCASCADE libraries.
-set (OpenCASCADE_LIBRARIES @OCCT_LIBRARIES@)
+# Hardcoded list of available OpenCASCADE libraries in correct linking order
+set (OpenCASCADE_LIBRARIES TKXDESTEP;TKXDEIGES;TKXCAF;TKVCAF;TKV3d;TKService;TKHLR;TKLCAF;TKCAF;TKCDF;TKSTEP;TKSTEPAttr;TKSTEP209;TKSTEPBase;TKIGES;TKBool;TKMesh;TKXSBase;TKShHealing;TKTopAlgo;TKPrim;TKBRep;TKGeomAlgo;TKGeomBase;TKBO;TKG3d;TKG2d;TKMath;TKernel)
 
 # OpenCASCADE global configuration options.
 set (OpenCASCADE_COMPILER          "@COMPILER@")
-- 
2.40.1

