From fbbfafacd8e1fef4e48e4363d7a9927225469dd1 Mon Sep 17 00:00:00 2001
From: Mathieu Westphal <mathieu.westphal@kitware.com>
Date: Sat, 20 Apr 2024 18:52:13 +0200
Subject: [PATCH] occt_remove_delete_fix_segfault

---
 src/Standard/Standard_Handle.hxx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Standard/Standard_Handle.hxx b/src/Standard/Standard_Handle.hxx
index db2d9431..4f2f1765 100644
--- a/src/Standard/Standard_Handle.hxx
+++ b/src/Standard/Standard_Handle.hxx
@@ -382,8 +382,8 @@ namespace opencascade {
     //! Decrement reference counter and if 0, destroy referred object
     void EndScope()
     {
-      if (entity != 0 && entity->DecrementRefCounter() == 0)
-        entity->Delete();
+//      if (entity != 0 && entity->DecrementRefCounter() == 0)
+//        entity->Delete();
       entity = 0;
     }
 
-- 
2.44.0

