diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1aef5d9..eb5d9ca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,6 +21,15 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 project(openvkl VERSION 1.0.1 LANGUAGES C CXX)
 
+#put rpath into install tree for ParaViewSB to fixup after
+set(CMAKE_SKIP_INSTALL_RPATH OFF)
+if (APPLE)
+  set(CMAKE_MACOSX_RPATH ON)
+  set(CMAKE_INSTALL_RPATH "@executable_path/" "@executable_path/../${CMAKE_INSTALL_LIBDIR}")
+else()
+  set(CMAKE_INSTALL_RPATH "\$ORIGIN:\$ORIGIN/../${CMAKE_INSTALL_LIBDIR}")
+endif()
+
 ## Add openvkl specific macros ##
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
