diff --git a/cmake/defaults/msvcdefaults.cmake b/cmake/defaults/msvcdefaults.cmake
index 1c4cb369d..8fa0e931b 100644
--- a/cmake/defaults/msvcdefaults.cmake
+++ b/cmake/defaults/msvcdefaults.cmake
@@ -124,13 +124,6 @@ _add_define("WIN32_LEAN_AND_MEAN")
 # for all translation units.
 set(_PXR_CXX_FLAGS "${_PXR_CXX_FLAGS} /bigobj")
 
-# Enable PDB generation.
-set(_PXR_CXX_FLAGS "${_PXR_CXX_FLAGS} /Zi")
-
-# Enable multiprocessor builds.
-set(_PXR_CXX_FLAGS "${_PXR_CXX_FLAGS} /MP")
-set(_PXR_CXX_FLAGS "${_PXR_CXX_FLAGS} /Gm-")
-
 # Ignore LNK4221.  This happens when making an archive with a object file
 # with no symbols in it.  We do this a lot because of a pattern of having
 # a C++ source file for many header-only facilities, e.g. tf/bitUtils.cpp.
