diff -u -Nr -U5 rkcommon-1.7.0/rkcommon/tasking/detail/tasking_system_init.cpp rkcommon-1.7.0.force-no-simd/rkcommon/tasking/detail/tasking_system_init.cpp
--- rkcommon-1.7.0/rkcommon/tasking/detail/tasking_system_init.cpp	2021-08-11 14:43:39.000000000 -0400
+++ rkcommon-1.7.0.force-no-simd/rkcommon/tasking/detail/tasking_system_init.cpp	2023-09-13 10:04:24.917412619 -0400
@@ -76,11 +76,11 @@
     static std::unique_ptr<tasking_system_handle> g_tasking_handle;
 
     void initTaskingSystem(int numThreads, bool flushDenormals)
     {
       if (flushDenormals) {
-        _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);
-        _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON);
+        /* _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON); */
+        /* _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON); */
       }
 
       g_tasking_handle = make_unique<tasking_system_handle>(numThreads);
     }
