diff --git a/ThirdParty/hdf5/vtkhdf5/config/ConfigureChecks.cmake b/ThirdParty/hdf5/vtkhdf5/config/ConfigureChecks.cmake
index b1d59ec1c3..4cb33143a6 100644
--- a/ThirdParty/hdf5/vtkhdf5/config/ConfigureChecks.cmake
+++ b/ThirdParty/hdf5/vtkhdf5/config/ConfigureChecks.cmake
@@ -449,6 +449,7 @@ CHECK_FUNCTION_EXISTS (qsort_s _HAVE_QSORT_S_TMP)
 if (_HAVE_QSORT_R_TMP OR _HAVE_QSORT_S_TMP)
   set (${HDF_PREFIX}_HAVE_QSORT_REENTRANT 1)
 endif ()
+set (${HDF_PREFIX}_HAVE_QSORT_REENTRANT 0)
 
 #-----------------------------------------------------------------------------
 # sigsetjmp is special; may actually be a macro
