From 5efde5c18fa5d976dc6de1198505ca1455d427da Mon Sep 17 00:00:00 2001
From: Mathieu Westphal <mathieu.westphal@kitware.com>
Date: Wed, 1 Apr 2026 16:26:07 +0200
Subject: [PATCH] apple-fix-database-location

---
 src/filemanager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/filemanager.cpp b/src/filemanager.cpp
index 69a1cf6a..a45fd5c6 100644
--- a/src/filemanager.cpp
+++ b/src/filemanager.cpp
@@ -1337,7 +1337,7 @@ static std::string pj_get_relative_share_proj_internal_no_check() {
     }
     out.resize(pos);
     // The initial path was something like foo/bin/libproj.so"
-    out += "/share/proj";
+    out += "/Resources";
     return out;
 #else
     return std::string();
-- 
2.53.0

